src/NUnitFramework/framework/Internal/Randomizer.cs

```diff
@@ -611,6 +611,25 @@ public decimal NextDecimal(decimal min, decimal max)
             return unchecked(raw % range + min);
         }
 
+        #endregion 
+
+        #region Guid
+
+        /// <summary>
+        /// Returns a random <see cref="Guid"/>.
```

Comment:
generates a valid type 4 guid? 

Put the bigger story in the documentation if necessary.